src/translators/sql_translator.py

```diff
@@ -828,14 +834,23 @@ def _get_where_clause(self, entity_ids, from_date, to_date, fiware_sp=None,
                     + fiware_sp + "($|/.*)'")
         else:
             # Match prefix of fiware service path
-            clauses.append(" " + prefix + FIWARE_SERVICEPATH + " = ''")
+            clauses.append(" " + prefix + FIWARE_SERVICEPATH + " = '' ")
+        attrs = ''
+        attrs_clauses = []
+        attrs_clauses.append(" and ")
+        if attr_names is not None:
```

Comment:
i am not sure the code added serve the purpose. the only place where you can compute the list of attributes that are linked to the time index, is where you compute the time_index. i.e.  `/src/reporter/timex.py`